docker/test/integration/steps/steps.py

```diff
@@ -621,6 +626,11 @@ def step_impl(context, num_flowfiles, duration):
     context.test.check_for_num_files_generated(int(num_flowfiles), timeparse(duration))
 
 
+@then("at least one flowfile is placed in the monitored directory in less than {duration}")
+def step_impl(context, duration):
+    context.test.check_for_num_files_generated(1, timeparse(duration))
```

Comment:
doesn't this look for exactly one file?

Reply:
you are right :+1: changed it to a range validator https://github.com/apache/nifi-minifi-cpp/pull/1294/commits/027fea1d65bd423940edd81d598cef68946e2202#diff-dce84c359f2f7128e501f5a322d8f6ac3325c2b471844f2c42cbef9abca58185R631